### DISCUSSION TITLE
how to use inner join

### DISCUSSION THREAD

**Discussion by takumiz19** [prisma/prisma#6814] [Q&A] [answered]
i have following schema.

```
model Genre {
  id                      BigInt                    @id @default(autoincrement())
  parentGenreId           BigInt?                   @map("parent_genre_id")
  parentGenre             ProductGenre?             @relation("ParentGenreChildGenres", fields: [parentGenreId], references: [id])
  childGenres             ProductGenre[]            @relation("ParentGenreChildGenres")
}
```

I wanted to do "inner join", so I wrote the following
```
const genres = await prisma.genre.findMany({
  include: {
    childGenres: true
  }
}
```
but result is like "left join"
how can i get inner join result??

**Comment by lyudad** [comment]
same issue

**Reply by jharrell** [reply, marked as answer]
Hi all, 

currently we don't support customizing the underlying joins via Prisma. This could be accomplished via a raw query. If you'd like, you can submit a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and describe your use case 😄 

#2836

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
